deploy/k8s/base/labelstudio/deployment.yml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: labelstudio
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: labelstudio
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        component: labelstudio
+    spec:
+      containers:
+        - name: labelstudio
+          image: heartexlabs/label-studio:1.0.2
+          imagePullPolicy: Always
+          stdin: true
+          tty: true
+          command: ["./deploy/wait-for-postgres.sh"]
+          args: ["labelstudio-postgres", "bash", "/label-studio/deploy/start_label_studio.sh"]
+          env:
+            - name: DJANGO_DB
+              value: default
+            - name: POSTGRE_NAME
+              value: postgres
+            - name: POSTGRE_USER
+              value: postgres
+            - name: POSTGRE_PASSWORD
```

Comment:
POSTGRE_PASSWORD not required here since you already set POSTGRES_HOST_AUTH_METHOD to trust

Reply:
Ha yes indeed! And it simplified things quite a lot since there's no need for a secret then, will remove it